endpoints/authentication/src/main.cpp

```diff
@@ -329,17 +329,16 @@ namespace irods::http::handler
 					// Not an error, likely to have id_token
 					// TODO: Consider handling bit flip cases
```

Comment:
Let's open an issue for this TODO and attach the issue number to it.
```
TODO(#NNN): Consider handling bit blip cases.
```

Reply:
Removing TODO, terrible comment, opening issue on 'improve OIDC client mode error handling in authenticate endpoint' instead.

Reply:
Created & removed.